Gemfile

```diff
@@ -1,14 +1,18 @@
 source 'https://rubygems.org'
 ruby "2.7.2"
 
-gem 'rails', '6.0.3.4'
+gem 'rails', '~> 6.1.0'
 
-gem 'pg'
+gem 'pg', '~> 1.1'
 
 gem 'coffee-script'
 gem 'uglifier'
-gem 'haml', '~> 5.1.2'
-gem 'sassc'
+gem 'haml', '~> 5.2.1'
+gem 'sass-rails', '>= 6'
```

Comment:
This gem has `sassc` as a dependency and hooks into asset pipeline more direclty